Artifacts/windows-vsts-build-agent/vsts-agent-install.ps1

```diff
@@ -0,0 +1,133 @@
+# Downloads the Visual Studio Online Build Agent, installs on the new machine, registers with the Visual
+# Studio Online account, and adds to the specified build agent pool
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    $vstsAccount,
+    
+    [Parameter(Mandatory=$true)]
+    $vstsUserPassword,
+    
+    [Parameter(Mandatory=$true)]
+    $agentName,
+    
+    [Parameter(Mandatory=$true)]
+    $poolname,
+    
+    [Parameter(Mandatory=$true)]
+    $driveLetter
+)
+
+Write-Verbose "Entering vsts-agent-install.ps1" -verbose
+
+if ($driveLetter.length -ne 1)
+{
+    Write-Error "The drive letter must be 1 character only; exiting."
+    exit -1
+}
+
+$vstsUser = "AzureDevTestLabs"
+
+$currentLocation = Split-Path -parent $MyInvocation.MyCommand.Definition
+Write-Verbose "Current folder: $currentLocation" -Verbose
+
+# Create a temporary directory to download from VSTS the agent package (agent.zip) to, and then launch the configuration.
+$agentTempFolderName = Join-Path $env:temp ([System.IO.Path]::GetRandomFileName())
+New-Item -ItemType Directory -Force -Path $agentTempFolderName
+Write-Verbose "Temporary agent download folder: $agentTempFolderName" -Verbose
+
+$serverUrl = "https://$vstsAccount.visualstudio.com"
```

Comment:
You should validate that this is truly only the account name and not the url here.